app/views/games/_jeopardy_table.html.haml

```diff
@@ -1,14 +1,12 @@
 %table.table.table-bordered.table-bolded#jeopardy-table
   %thead
     %tr
-      - @category_ids.each do |category|
-        %th.text-center{ style: "width:#{heading_width(@category_ids)}%;" }=category_ids_to_names(category, @categories)
+      %th.text-center{ style: "width: 25%;" }= t('game.show.challenges_table.sponsor_header')
+      %th.text-center{ style: "width: 50%;" }= t('game.show.challenges_table.name_header')
+      %th.text-center{ style: "width: 15%;" }= t('game.show.challenges_table.category_header')
+      %th.text-center{ style: "width: 10%;" }= t('game.show.challenges_table.points_header')
```

Comment:
I would really prefer to keep the jeopardy board looking the exact same here. My vision was the jeopardy board would stay the same but the challenge itself would show the sponsor on the challenge show page. We could also add a new "Hackathon" board layout down the road that shows each challenge as title, short description, and sponsor all on one page.